server/accounts.go

```diff
@@ -878,7 +878,9 @@ func (a *Account) randomClient() *client {
 	}
 	var c *client
 	for c = range a.clients {
-		break
+		if c.acc == a {
```

Comment:
But now you may get DATA RACE reports because c.acc is sometimes changed. I wonder if we should not rework that whole sending subs through route.

Reply:
I wanted to see if travis picked it up, I know we do things around being able to access c.acc without a lock.
Agree we should on routes, I even have a comment there about it ;)

Reply:
I am seeing another test failure. Looking into that one now.